### DISCUSSION TITLE
Very minor and obvious typo in docs

### DISCUSSION THREAD

**Discussion by polebridge** [micropython/micropython#16020] [General]
Is this the place to mention a typo in the docs? If not, where should i go?
at https://docs.micropython.org/en/latest/esp8266/tutorial/ssd1306.html
display.rect(10, 10, 107, 43, 1)        # draw a rectangle outline 10,10 to 117,53, colour=1
display.fill_rect(10, 10, 107, 43, 1)   # draw a solid rectangle 10,10 to 117,53, colour=1
It appears that 53 should be 43, or vv
Also, perhaps color has an extra 'u'? :)
Thanks

**Comment by davefes** [comment]
Probably here:
https://github.com/micropython/micropython/issues
there is a Documentation tag

**Comment by projectgus** [comment]
@polebridge If you're keen then please submit a Pull Request to fix the comment. For this kind of edit you can even do it from the GitHub web interface. The source doc is https://github.com/micropython/micropython/blob/master/docs/esp8266/tutorial/ssd1306.rst

> Also, perhaps color has an extra 'u'? :)

MicroPython was started in Australia and only a few of the contributors are Americans, so the 'u' probably stays. :laughing:

**Reply by mattytrentini** [reply]
> so the 'u' probably stays. 😆

Yes, that's a feature not a defect 😜

**Comment by corranwebster** [comment]
I don't think that this is a bug: the `rect` and `fill_rect` commands expect arguments of the form `x, y, width, height`, not `x0, y0, x1, y1`, so the rectangles in the examples have a width of 43, meaning the right-hand edge is at x1 = 10 + 43 = 53 (similarly the bottom edge is at y1 = 10 + 107 = 117)

**Reply by projectgus** [reply]
Ah, well spotted @corranwebster!

**Reply by polebridge** [reply]
Yes, i'll have to improve my reading for comprehension.  Thanks.
